assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -155,7 +155,9 @@ const ContextMenu = ( props ) => {
 
 		const rootClientId = getBlockRootClientId( clientId );
 		const clonedBlock = cloneBlock( block );
-		insertBlock( clonedBlock, null, rootClientId );
+		const selectBlock = ( clonedBlock.name === 'amp/amp-story-page' ) ? '' : rootClientId;
```

Comment:
Rename to `parentBlock`.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3537/commits/bbac22795430583fc9ba28c9e73953e90dedfcc0